docs/PeopleGroup/Offboarding.mdx

```diff
@@ -9,22 +9,10 @@ import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
 <MaintainerTag maintainerEmails={["akshat.mittal@openenergytransition.org", "fabrizio.finozzi@openenergytransition.org"]} />
 
 # Offboarding
-this should be same as sidebar_label and folder name
 
 <ControlledDocBanner />
 
-should be present on every controlled document.
-
-## Sub Heading
-
-At our company, we are committed to providing a safe and inclusive workplace for all employees. This policy outlines our expectations for behavior and conduct in the workplace.
-
-## Another Sub Heading
-
-All employees are expected to adhere to the following code of conduct:
-
-- Treat all colleagues with respect and professionalism
-- Avoid any behavior that could be considered discriminatory or harassing
-- Maintain confidentiality of company and client information
-- Follow all company policies and procedures
-- Report any violations of this code of conduct to management
+The manager of the person leaving OET must ensure that by the end of their last day:
+- The person's OET Google account (email) is deactivated (ask TODO if you need support with this)
```

Comment:
Maybe Victor Martinez is the right contact person for deactivating email accounts?

Reply:
@VictorAMartinez can I put your name here? 🙏